luigi/scheduler.py

```diff
@@ -870,6 +871,7 @@ def _serialize_task(self, task_id, include_deps=True, deps=None):
             'priority': task.priority,
             'resources': task.resources,
             'tracking_url': getattr(task, "tracking_url", None),
+            'status_message': task.status_message
```

Comment:
I think you can copy the line above. Make it like 

`getattr(task, "status_message", None)` instead of `task.status_message`. Then a couple of months later we're changing it back to `task.status_message`. Do you see how it'll work? Do you mind submitting a PR (you can use the online editor)